src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    static ArrayList<Task> list = new ArrayList<Task>();
```

Comment:
You can use a better name (according to prof) - I think you should use a better name that's more specific, such as tasksList or something like that